test/image/mocks/gl2d_connect_gaps.json

```diff
@@ -12,10 +12,11 @@
       "y": [-2, -2, null, 2, 2],
       "connectgaps": true,
       "mode": "lines",
-      "type": "scattergl"
+      "type": "scattergl",
+      "opacity": 0.4
     }
   ],
   "layout": {
-    "title": "Connect gaps test"
+    "title": "Connect gaps + line opacity test"
```

Comment:
OK I see this is one type of new coverage. Btw. indeed the image tests are rather expensive, around 1s each.